google-cloud-tools-plugin/src/com/google/gct/idea/debugger/ui/CloudAttachDialog.java

```diff
@@ -177,7 +176,7 @@ protected void doOKAction() {
       UsageTrackerProvider.getInstance()
           .trackEvent(GctTracking.CATEGORY, GctTracking.CLOUD_DEBUGGER, "start.session", null);
       // TODO : add source context tracking info
-      if (mySyncStashCheckbox.isSelected()) {
+      if (mySyncStashCheckbox.isVisible() && mySyncStashCheckbox.isSelected()) {
```

Comment:
oh this was from testing stuff and I didn't take it out, good catch.
I do think a checkbox can be invisible and checked though...